test/verify/check-system-services

```diff
@@ -630,10 +630,18 @@ WantedBy=multi-user.target
         b.wait_not_present("#condition")
 
     def testRelationships(self):
+        self._testRelationships()
+
+    def testRelationshipsSession(self):
+        self._testRelationships(True)
+
+    def _testRelationships(self, user=False):
         m = self.machine
         b = self.browser
 
-        self.write_file("/etc/systemd/system/test-a.service",
+        path = "/etc/systemd/user" if user else "/etc/systemd/system"
```

Comment:
s/path/unit_dir/ ? The scope of "path" is quite large, so the variable name should be self-describing.